### DISCUSSION TITLE
Ansicht "Vorhersage vom DWD"

### DISCUSSION THREAD

**Discussion by hoetzgit** [Daveiano/weewx-wdc#85] [Ideas]
Hallo David,

ich möchte Dich nicht überstrapazieren, aber was hältst Du davon, die Vorhersage etwas zu strukturieren? (Muster: DWD Seite)

Beispiel:

![image](https://user-images.githubusercontent.com/41108116/196924742-e3f9f157-58ac-442f-93e7-e49613076cf8.png)

**Comment by Daveiano** [comment]
Gute Idee! Kommt ins nächste Release, siehe #86
